src/Pages/ViewLog.php

```diff
@@ -14,13 +14,14 @@ class ViewLog extends Page
 {
     use HasSecurity;
 
-    protected static string $view = 'filament-laravel-log::view-log';
+    public static string $view = 'filament-laravel-log::view-log';
 
     public ?string $logFile = null;
 
     public function refreshContent(): void
     {
-        $this->dispatchBrowserEvent('logContentUpdated', ['content' => $this->readLog()]);
+        $content = $this->readLog();
+        $this->dispatch('logContentUpdated', ['content' => $content]);
```

Comment:
$this->dispatch('logContentUpdated', ['content' => $this->readLog()]);

just one line